core/google/cloud/_helpers.py

```diff
@@ -34,7 +34,7 @@
     import grpc
     from google.auth.transport.grpc import (
         AuthMetadataPlugin)  # pragma: NO COVER
-except ImportError:
+except ImportError:  # pragma: NO COVER
```

Comment:
No, because-

case 1: grpc isn't available, `import grpc` raises `ImportError` and the second import is never executed.
case 2: grpc is available, both imports succeed and the `except` is never executed.

To remove the one above, I'd need to make a case where `import grpc` works but `import google.auth.transport.grpc` doesn't, which for all practical purposes is meaningless.